src/components/TextArea/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import _get from 'lodash/get';
+
+import { Answer } from 'types/answer';
+
+export const textAreaDataTestIds = {
+  base: 'text-area__base',
+};
+
+type TextAreaProps = {
+  questionId: string;
+  items: Answer[];
```

Comment:
Do we need these values? The default value I mentioned is the value of the TextArea 😅 So we can show some value that we need to show by default: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea2
If in your case, you don't need the default value, I think you also need a default placeholder, right?
So: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_textarea2

Reply:
@hoangmirs updated in [818a81c](https://github.com/manh-t/react-survey/pull/76/commits/818a81c509fca09146d82d79dcbc14801e1c7da2)